integration_tests/sdk/conftest.py

```diff
@@ -36,6 +30,16 @@ def fetch_flags(pytestconfig):
     yield
 
 
+def all_data_integration_names() -> List[str]:
+    """We currently only support the demo database, but this can eventually be configured arbitrarily."""
+    return ["aqueduct_demo"]
+
+
+@pytest.fixture(scope="session", params=all_data_integration_names())
```

Comment:
How is the `params` parameter used here? I'm not too familiar with the usage of this within fixtures.

Reply:
it's referenced by `request.param` in the body